VERSION

```diff
@@ -1 +1 @@
-2.37.0-rc.0
+2.37.0-rc.1
```

Comment:
is it normal that you have a "." between `rc` and `1` ? On other rc we don't have that

Reply:
2.34.0-rc.0
2.33.0-rc.0
2.32.0-rc.0
2.31.0-rc.0
2.30.0-rc.0

2.35 and 2.36 are the different ones